02_assignments/assignment-1.ipynb

```diff
@@ -0,0 +1,74 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "True\n"
+     ]
+    }
+   ],
+   "source": [
+    "def anagram_checker(word_a, word_b):\n",
+    "    first_word = sorted(word_a.lower())\n",
+    "    second_word = sorted(word_b.lower())\n",
+    "\n",
+    "    return first_word == second_word\n",
+    "\n",
+    "print(anagram_checker(\"night\", \"Thing\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def anagram_checker_case_sensitive(word_a, word_b, is_case_sensitive):\n",
+    "    if is_case_sensitive:\n",
+    "         first_word = sorted(word_a)\n",
+    "         second_word = sorted(word_b)\n",
+    "    else:\n",
+    "         first_word = sorted(word_a.lower())\n",
+    "         second_word = sorted(word_b.lower())\n",
+    "\n",
+    "         return first_word == second_word\n",
```

Comment:
second commit I fixed this mistake.
I saw that the return statement was still inside of if else. 
In the second commit i fix, but not sure it is visible correct